src/angularjs/src/app/places/compare/compare.controller.js

```diff
@@ -31,22 +31,23 @@
             });
 
             // first request score metadata
-            promises.unshift(ScoreMetadata.query().$promise);
+            promises.unshift(ScoreMetadata.query());
 
             // do not display any place until all places have been retrieved
             $q.all(promises).then(function(results) {
                 // first element is the metadata; rest are the places
-                var metadata = _.reduce(_.head(results), function (result, score) {
-                    result[score.name] = {
-                        label: score.label,
-                        category: score.category,
-                        description: score.description
-                    };
-                    return result;
-                }, {});
-                ctl.metadata = metadata;
+                ctl.metadata = _.head(results);
 
                 ctl.places = _.drop(results);
+                _.forEach(ctl.places, function (place) {
+                    place.scores = _.reduce(place.scores, function (result, score) {
+                        result[score.metric] = {
+                            score: score.score,
+                            score_normalized: score.score_normalized
+                        }
+                        return result;
+                    }, {});
+                })
```

Comment:
Whoops missed keyBy. I'll switch to that.

Yeah, I'll look at putting this change in the service.